dsf-gdb/org.eclipse.cdt.dsf.gdb/src/org/eclipse/cdt/dsf/gdb/launching/FinalLaunchSequence.java

```diff
@@ -552,39 +546,6 @@ protected void handleError() {
 		rm.done();
 	}
 
-	private static final String INVALID = "invalid"; //$NON-NLS-1$
-
-	/**
-	 * If we are dealing with a remote-attach debugging session, connect to the target.
-	 * @since 4.0
-	 */
-	@Execute
-	public void stepRemoteConnection(final RequestMonitor rm) {
```

Comment:
This is now API and can't be removed this simply.

Reply:
yup! what should be the way forward?

Reply:
Now that I have reproduced I will start trying to figure out what workarounds we need to do about this.

API compatibility errors is just about binary compatibility, sometimes we make semantic changes that doesn't break API, but is essentially breaking change in behaviour.

Anyway, I'll try to provide a concrete suggestion.

Reply:
kept the API wit deprecated annotation and its no more called..